python/cuml/common/handle.pyx

```diff
@@ -48,8 +49,8 @@ cdef class Handle:
 
         # call ML algos here
 
-        # final synchronization of all work launched/dependent on this stream
-        stream.sync()
+        # final sync of all work launched in the stream of this handle
```

Comment:
Regarding the comment my remark I made on `base.pyx` also applies here.

Reply:
Done.